### ISSUE TITLE
Bad Escape - rsp train

### ISSUE THREAD

**Issue by tgoers** [datapink/robosat.pink#85]
**I am attempting to train my model (rsp train) and I am getting errors I am using the provided config file and keep getting the error: "ERROR: bad escape \i at position 11". I am running Python 3.7.5.**

`(base) C:\Users\trist\OneDrive\Desktop>rsp train --config ds/config.toml --epochs 10 --ts 1024,1024 ds ds/model
RoboSat.pink - training on CPU, with 4 workers - (Torch:1.3.1)

==========================================================
WARNING: Are you -really- sure about not training on GPU ?
==========================================================

--- Input tensor from Dataset: ds ---
Channel 1:               images[band: 1]
Channel 2:               images[band: 2]
Channel 3:               images[band: 3]
--- Output Classes ---
Class 0:                 Background
Class 1:                 Building
--- Hyper Parameters ---
nn                       Albunet
loss                     Lovasz
bs                       4
lr                       2.5e-05
ts                       (1024, 1024)
ms                       16
loader                   SemSegTiles
da                       Strong
metrics                  ['iou', 'mcc']
pretrained               True
dap                      1.0

ERROR: bad escape \i at position 11`

**If I change the name field to something else, then I get another bad escape error:**

`(base) C:\Users\trist\OneDrive\Desktop>rsp train --config ds/config.toml --epochs 10 --ts 1024,1024 ds ds/model
RoboSat.pink - training on CPU, with 4 workers - (Torch:1.3.1)

==========================================================
WARNING: Are you -really- sure about not training on GPU ?
==========================================================

--- Input tensor from Dataset: ds ---
Channel 1:               images_afnajfn[band: 1]
Channel 2:               images_afnajfn[band: 2]
Channel 3:               images_afnajfn[band: 3]
--- Output Classes ---
Class 0:                 Background
Class 1:                 Building
--- Hyper Parameters ---
nn                       Albunet
loss                     Lovasz
bs                       4
lr                       2.5e-05
ts                       (1024, 1024)
ms                       16
loader                   SemSegTiles
da                       Strong
metrics                  ['iou', 'mcc']
pretrained               True
dap                      1.0

ERROR: bad escape \l at position 11`

**Comment by ocourtin** [comment]
Hi,

Looks like you are trying to run RoboSat.pink on Windows. 
And obviously it's not supported.
You have to move to Linux.

And using an NVIDIA GPU is also hardly recommanded.
If you've got neither. You can also rent one (on OVHCloud or AWS for instance...)


HTH,

**Comment by tgoers** [comment]
> Hi,
> 
> Looks like you are trying to run RoboSat.pink on Windows.
> And obviously it's not supported.
> You have to move to Linux.
> 
> And using an NVIDIA GPU is also hardly recommanded.
> If you've got neither. You can also rent one (on OVHCloud or AWS for instance...)
> 
> HTH,

Oh, well everything up to this point has been working.

**Comment by ocourtin** [comment]
Indeed DataPreparation steps don't need huge computation, as so are performed on CPU.
But as documented `rsp train` and `rsp predict` use GPU if available.
